source/code/source/playfab/PlayFabCurlHttpPlugin.cpp

```diff
@@ -6,6 +6,8 @@
 
 #include <stdexcept>
 
+#define _countof(array) (sizeof(array) / sizeof(array[0]))
```

Comment:
needs parentheses around the array part of array[0]

sizeof((array)[0])

have you considered using the countof helper we define in BumblelionArrayHelpers.h instead? It will prevent you from accidentally using this on a pointer type rather than an array type.

Reply:
This was copy-pasted from the official doc:
https://docs.microsoft.com/en-us/cpp/c-runtime-library/reference/countof-macro?view=vs-2019
I'm willing to use whatever seems best though.
I'll go grab the one in BumblelionArrayHelpers.h